Core/Systems/Questing/QuestHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using PathOfTerraria.Core.Systems.Affixes;
+using PathOfTerraria.Core.Systems.Questing.Quests.TestQuest;
+using PathOfTerraria.Core.Systems.Questing.QuestStepTypes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Terraria.GameContent.UI.Chat;
+using Terraria.ModLoader.IO;
+
+namespace PathOfTerraria.Core.Systems.Questing;
+internal class QuestHandler : ModPlayer
+{
+	// need a list of what npcs start what quests
+	private readonly List<Quest> _enabledQuests = [];
+
+	public void RestartQuestTest()
+	{
+		_enabledQuests.Clear();
+		Quest quest = new TestQuest();
+
+		quest.StartQuest(Player);
+
+		_enabledQuests.Add(quest);
+	}
+
+	public override void PostUpdate()
+	{
+		// _enabledQuests.ForEach(q => Console.WriteLine(q.CurrentQuestString()));
```

Comment:
Temp thing?

Reply:
If you uncomment it it prints the progress of all quests; as we dont have ui for it yet it will just be a temp ting so that we can have an idea of whats going on (if one decides to work on quests in any way shape or form).

So yes.